src/patternfly/components/Table/table-grid.scss

```diff
@@ -332,7 +331,7 @@
     --pf-c-table-cell--PaddingRight: 0;
     --pf-c-table-cell--PaddingLeft: 0;
 
-    display: block;
+    display: initial;
```

Comment:
This will set it to `display: inline;`, which works but I wonder if there's a better way to fix it since that element will no longer behave as a block level element (and I think it should).

It looks like the issue is due to the stacking context of `.pf-c-table__expandable-row` being placed above the blue border because the expandable row has `position: relative`. It doesn't look like `.pf-c-table__expandable-row` needs `position: relative;` in a grid (or desktop?) layout, so one option to could be to remove it.

Or another option could be - since the blue border is drawn with `.pf-m-grid-lg.pf-c-table tbody:before`, we could just switch that to `.pf-m-grid-lg.pf-c-table tbody:after` to fix the stacking context.

Reply:
Those are great ideas thanks, I went with changing the blue border to use `::after` instead of `::before`